index.html

```diff
@@ -1,52 +1,352 @@
             <h1 class="headline-primary">Hey there. <br>
                 <span class="headline-primary">I'm Graciano.</span> <br>
-                <span class="headline-secondary">I'm a software developer</span>     
+                <span class="headline-secondary">I'm a software developer</span>
             </h1>
-         </div>
-         <div class="headline-content">
-            <p>
-                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,         
-                  don't hesitate to contact me.
-            </p>
-         </div> 
-
-         <ul class="socialmedia-links">
-            <li><a href="https://twitter.com/GracianoSoft"><img src="assets/img/twitter.svg" alt=""></a></li>
-            <li><a href="https://www.linkedin.com/in/graciano-manuel-henrique-175527221/"><img src="assets/img/Linkedin icon.svg" alt=""></a></li>
-            <li><a href="https://medium.com/@gracianomanuelhenrique"><img src="assets/img/medium.svg" alt=""></a></li>
-            <li><a href=" https://github.com/Graciano1997"><img src="assets/img/github.svg" alt=""></a></li>
-            <li><a href="#"><img src="assets/img/angellist.svg" alt=""></a></li>
-            
-         </ul> 
-        
-     
-        
+            <div class="headline-content">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don't hesitate to contact me.
+                </p>
+            </div>
+
+            <ul class="socialmedia-links">
+                <li><a href="https://twitter.com/GracianoSoft"><img src="assets/img/twitter.svg" alt=""></a></li>
+                <li><a href="https://www.linkedin.com/in/graciano-manuel-henrique-175527221/"><img
+                            src="assets/img/Linkedin icon.svg" alt=""></a></li>
+                <li><a href="https://medium.com/@gracianomanuelhenrique"><img src="assets/img/medium.svg" alt=""></a>
+                </li>
+                <li><a href=" https://github.com/Graciano1997"><img src="assets/img/github.svg" alt=""></a></li>
+                <li><a href="#"><img src="assets/img/angellist.svg" alt=""></a></li>
+            </ul>
+            <div class="btn-hover"><img src="assets/img/see.svg" alt="" srcset=""></div>
+        </section>
+
+        <section class="works-section">
+            <div class="headline-container">
+                <h1 class="headline-primary">My Recent <br>
+                    <span class="headline-primary">Works</span> <br>
+                </h1>
+                <hr class="indicator">
+            </div>
+
+            <div class="box-card">
+                <div class="cards">
+                    <div class="card-headerbg">
+                    </div>
+                    <div class="card">
```

Comment:
- [ ] kindly consider aligning elements in each card with `flex` instead of `grid` because it's required to use flex on a single card and give the whole section grid display rather than doing the inverse check again on the below requirement for more clarification 
<img width="1262" alt="Screen Shot 2023-06-05 at 14 32 57" src="https://github.com/Graciano1997/portofolio-graciano/assets/60146030/d5930854-04f6-4ac8-ac5a-22788ed844e1">